.actions/setup_tools.py

```diff
@@ -394,74 +178,63 @@ def parse_version_from_file(pkg_root: str) -> str:
     return ver
 
 
-def prune_duplicate_lines(body):
-    body_ = []
-    # drop duplicated lines
-    for ln in body:
-        if ln.lstrip() not in body_ or ln.lstrip() in (")", ""):
-            body_.append(ln)
-    return body_
-
-
-def create_meta_package(src_folder: str, pkg_name: str = "pytorch_lightning", lit_name: str = "pytorch"):
-    """Parse the real python package and for each module create a mirroe version with repalcing all function and
-    class implementations by cross-imports to the true package.
+def _replace_imports_in_file(lines: List[str], pkg_lut: Dict[str, str]) -> List[str]:
+    """REplace imports of standlone package to lightning.
+
+    >>> lns = ["lightning_app",
+    ...        "delete_cloud_lightning_apps",
+    ...        "from lightning_app import",
+    ...        "lightning_apps = []",
+    ...        "lightning_app is ours",
+    ...        "def _lightning_app():",
+    ...        ":class:`~lightning_app.core.flow.LightningFlow`"]
+    >>> from pprint import pprint
+    >>> pprint(_replace_imports_in_file(lns, {"app": "lightning_app"}))
+    ['lightning.app',
+     'delete_cloud_lightning_apps',
+     'from lightning.app import',
+     'lightning_apps = []',
+     'lightning.app is ours',
+     'def _lightning_app():',
+     ':class:`~lightning.app.core.flow.LightningFlow`']
+    """
+    for n2, n1 in pkg_lut.items():
+        for i, ln in enumerate(lines):
+            lines[i] = re.sub(rf"([^_]|^){n1}([^_\w]|$)", rf"\1lightning.{n2}\2", ln)
```

Comment:
can you add a comment for what this regex does so that we don't need to look it up every time?

Reply:
I belive it is covered by the example in docstring :otter: